src/main/java/frc/robot/subsystems/BuddyBalanceSubsystem.java

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.networktables.GenericEntry;
+import edu.wpi.first.util.datalog.DataLog;
+import edu.wpi.first.util.datalog.DoubleLogEntry;
+import edu.wpi.first.wpilibj.DataLogManager;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.hardware.MotorController;
+import frc.robot.hardware.MotorController.MotorConfig;
+import frc.robot.OI;
+import frc.robot.classes.TunableNumber;
+
+import javax.management.relation.Relation;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkMaxPIDController;
+
+public class BuddyBalanceSubsystem extends SubsystemBase { // TODO: Find a permanent name for buddy balance subsystem - BBalance is ambiguous
+  public static double buddyBalanceRefPointDocked;
+  public static double buddyBalanceRefPointBalanced;
+  public static double buddyBalanceRefPointDeployed;
```

Comment:
Comment what each position is used for

Reply:
I didn't make the reference constants final because I was planning to tune them with TunableNumbers to save robot time
They will be final when they are tuned correctly